api/bootstrap.proto

```diff
@@ -213,3 +217,10 @@ message RateLimitServiceConfig {
   // limit service requests.
   string cluster_name = 1 [(validate.rules).string.min_bytes = 1];
 }
+
+// MetricService Configuration structure.
+message MetricsServiceConfig {
+  // The name of the upstream cluster that hosts the metrics service. The cluster must be
+  // configured in the cluster manager.
+  string cluster_name = 1 [(validate.rules).string.min_bytes = 1];
```

Comment:
This looks like a generic gRPC client config. Do we really need such specific proto definition? I think the following code would work just find.

```
GrpcSettings authz_service = 1;
GrpcSettings logging_service = 2;
GrpcSettings metrics_service = 3;
```
We are going to need cluster name, timeout, retries, fail open/close, etc for each gRPC service. It would be redundant to repeat the definition everywhere.

For the product I am currently running, it connects to 20+ other services. We cannot afford to have one config for each service.

Reply:
Yup this can be switched to `GrpcCluster`. @ramaraochavali can you fix this and also the access log config? Thanks.

Reply:
Nevermind I will fix the access log one and do the envoy code change also.

Reply:
No problem. I will use this PR to make the change for metrics proto